source/_docs/lockr.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Using Lockr To Secure and Manage API and Encryption Keys With WordPress and Drupal
+type: guide
+permalink: docs/guides/:basename/
+description: Detailed information on how to setup and use Lockr in your WordPress and Drupal site.
+categories: [sites]
+tags: [code]
+contributors:
+  - cteitzel
+date: 8/11/2016
+---
+## What is Lockr
+
+[Lockr](https://lockr.io) is a simple-to-use plugin for WordPress or Drupal to manage your site’s API and Encryption keys in a secure offsite hosted environment. Lockr removes the key from your site code and database and stores it in a secure and certified key manager. When your site needs the key to be used in an encryption/decryption or API request, Lockr will use the Pantheon server’s certificate to authenticate on your behalf and release the key. 
+
+Lockr allows site administrators the ability to take control of how and where their sensitive keys are stored, thus improving the overall security of the site and allows it to meet specific regulatory and compliance requirements around key management. Lockr also separately stores development and production specific keys to create an additional layer of security between environments.
+
+## Why Lockr?
+
+By combining a simple to use developer interface with a managed scalable cloud key management system, Lockr allows applications of all sizes to meet industry standards for key management. Unlike other key managers, Lockr offers a "white glove service" that provides additional layers of security and system monitoring, no ongoing maintenance and continuous development for integrations with your favorite modules and plugins. Offsite API & encryption key management delivers best-practice security to help sites comply with HIPAA, FERPA, and FISMA among others. Development use of Lockr in the Dev, Test and Multi-Dev environments is always free.  
+
+## WordPress Installation
+
+Lockr is automatically configured to secure API keys for numerous 3rd party plugins for seamless integration and securing of your keys. 
+Visit our [Github page](https://github.com/lockr/lockr-patches/tree/wp) for a list of plugins that can be automatically patched.
+
+### Install through the Administration Interface
+
+The official [Lockr Plugin](https://wordpress.org/plugins/lockr) makes it a breeze to get started and quickly secure your keys. Install and activate the latest release through the WordPress dashboard or place it it in /wp-content/plugins directory and activate via the dashboard.
+
+1. Install through the WordPress plugin screen directly or upload the Lockr package to `/wp-content/plugins/` directory. 
+2. Activate Lockr through the “Plugins” section in your WordPress backend  (ex. `http://my-site.pantheonsite.io/wp-admin/plugins.php` )
```

Comment:
_Align installation steps with other docs (e.g. [Configuring Environment Indicators](https://pantheon.io/docs/environment-indicator/#wordpress-pantheon-hud))_

1. [Set the connection mode to SFTP](/docs/sftp) for the Dev or Multidev environment via the Pantheon Dashboard or with [Terminus](/docs/terminus):
 
 ```nohighlight
 terminus site set-connection-mode --mode=sftp
 ```
 
2. Install and activate the [Lockr](https://wordpress.org/plugins/lockr) plugin from within the Dev or Multidev environment's WordPress Dashboard (`/wp-admin/plugin-install.php?tab=search&s=pantheon+hud`) or with [Terminus](/docs/terminus):
 
 ```nohighlight
 terminus wp 'plugin install lockr --activate'
 ```